build/scripts/build.ps1

```diff
@@ -362,6 +363,60 @@ function Build-InsertionItems() {
     }
 }
 
+function Build-Installer () {
+    #  Copying Artifacts
+    $installerDir = Join-Path $configDir "Installer"
+    if (-not (Test-Path $installerDir)) {
+        New-Item -ItemType Directory -Force -Path $installerDir
+    }
+    
+    $intermidateDirectory = Join-Path $env:TEMP "InstallerTemp"
+    if(Test-Path $intermidateDirectory)
+    {
+        Remove-Item -Path $intermidateDirectory -Recurse -Force
+    }
+    New-Item -ItemType Directory -Force -Path $intermidateDirectory
+
+    ## Copying VsixExpInstaller.exe
+    $vsixExpInstallerDir = Get-PackageDir "RoslynTools.Microsoft.VSIXExpInstaller"
+    $vsixExpInstallerExe = Join-Path $vsixExpInstallerDir "tools\VsixExpInstaller.exe"
+    $vsixExpInstallerExeDestination = Join-Path $intermidateDirectory "tools\vsixexpinstaller"
+    if (-not (Test-Path $vsixExpInstallerExeDestination)) {
+        New-Item -ItemType Directory -Force -Path $vsixExpInstallerExeDestination
+    }
+    Copy-Item -Path $vsixExpInstallerExe -Destination $vsixExpInstallerExeDestination
+    
+    ## Copying VsWhere.exe
+    $vswhere = Join-Path (Ensure-BasicTool "vswhere") "tools\vswhere.exe"
+    $vswhereDestination = Join-Path $intermidateDirectory "tools\vswhere"
+    if (-not (Test-Path $vswhereDestination)) {
+        New-Item -ItemType Directory -Force -Path $vswhereDestination
+    }
+    Copy-Item -Path $vswhere -Destination $vswhereDestination
```

Comment:
Think we need to copy the entire vswhere directory vs. the single exe. Today it's a standalone exe but if they ever change that we'd be broken here.